.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
   - TEST_GROUP="cucumber -f progress -r features features/other_b"
   - TEST_GROUP="cucumber -f progress -r features features/prompt_memes_a"
   - TEST_GROUP="cucumber -f progress -r features features/prompt_memes_b"
+  - TEST_GROUP="cucumber -f progress -r features features/prompt_memes_c"
```

Comment:
Ditto to this--I think it's a perfectly good idea to break up the prompt memes tests (they really need a total overhaul) but doesn't seem to belong in this pullreq? Maybe just move them into a separate one that can be merged fast since it's just changing tests.